Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
diff --git a/output/Radio Group with Accessible Description/1/18d5f8a6-14a3-4ead-ae63-572bd1869b49.html b/output/Radio Group with Accessible Description/1/18d5f8a6-14a3-4ead-ae63-572bd1869b49.html
index dd44aa3..13551ed 100644
--- a/output/Radio Group with Accessible Description/1/18d5f8a6-14a3-4ead-ae63-572bd1869b49.html	
+++ b/output/Radio Group with Accessible Description/1/18d5f8a6-14a3-4ead-ae63-572bd1869b49.html	
@@ -1,9 +1,13 @@
-<div>
-  <fieldset>
-    <legend>Choose a theme</legend>
-    <label><input type="radio" name="theme" value="light"> Light</label>
-    <label><input type="radio" name="theme" value="dark"> Dark</label>
-    <label><input type="radio" name="theme" value="high-contrast"> High Contrast</label>
-  </fieldset>
-  <small>You can change this later</small>
-</div>
\ No newline at end of file
+<fieldset aria-describedby="theme-description">
+  <legend>Choose a theme</legend>
+  <small id="theme-description">You can change this later</small>
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light"> Light
+  </label>
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark"> Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast"> High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
